crates/nargo_cli/src/cli/prove_cmd.rs

```diff
@@ -171,7 +162,7 @@ pub(crate) fn prove_with_path<B: Backend, P: AsRef<Path>>(
     }
 
     let proof_path = if let Some(proof_name) = proof_name {
-        Some(save_proof_to_dir(&proof, &proof_name, proof_dir)?)
+        Some(save_proof_to_dir(&proof, proof_name, proof_dir)?)
```

Comment:
This results in all packages writing to the same file in workspaces so we only get the final package's proof.

Reply:
We dropped `proof_name` and are just using package name now. We also are always saving the proof to the directory as we collect feedback on if printing on stdout is valuable. I will open a new issue to collect feedback and then resolve this.